### DISCUSSION TITLE
How to automatically download confirmations from EQSL.CC?

### DISCUSSION THREAD

**Discussion by MikleBas** [magicbug/Cloudlog#2240] [Q&A] [answered]
LOTW does it perfectly by itself on the crown, but eqsl always has to be in manual mode, can this action be automated somehow? At least every 15 minutes?

PS my log works in conjunction with GridTracker - and it probably (or Cloudlog) throws the connections made outside, but it has to be returned manually, what would you advise?
Thank you!

de UD3AYQ
73!

**Comment by AndreasK79** [comment, marked as answer]
Hi, a cron job was added a little while back for this purpose:

# Upload/download QSOs to/from Eqsl (ignore cron job if this integration is not required)
9 */6 * * * curl --silent https://<URL-To-Cloudlog>/index.php/eqsl/sync &>/dev/null

**Reply by MikleBas** [reply]
Hello!
Everything works great, thanks! :)
73!
